### PR TITLE
Crop out secpk256k1

### PR DESCRIPTION
**Pull request by VoR0220**

### CODE REVIEW

File: crypto/key.go [hunk 1]
```diff
@@ -78,8 +77,8 @@ type CurveType uint8
 
 func (k CurveType) String() string {
 	switch k {
-	case CurveTypeSecp256k1:
-		return "secp256k1"
+	/*case CurveTypeSecp256k1:
+		return "secp256k1"*/
```
Comment (pietv) [inline crypto/key.go]: can you remove the code you cropping altogether?

**Comment by pietv** [comment]
LGTM (please squash the "forget to get rid of..." commits)

File: crypto/key.go (hunk 1 above)
Comment (benjaminbollen) [inline crypto/key.go]: no, this setup that was there in eris-keys was solid; this is not what I asked you to do.

**Comment by benjaminbollen** [comment]
This should not be merged. This is not what needs to be done; neither was RJ asked to do this.

**Comment by benjaminbollen** [comment]
the existing framework for secp256k1 should be left as it was, or fixed; not removed.

**Pull request closed without merging**